### PR TITLE
Got to everything I could!

### PR DESCRIPTION
**Pull request by rowlandhill**

### CODE REVIEW

**Comment by rowlandhill** [comment]
3 of 5 fist

File: diagnostic.md [hunk 1]
```diff
@@ -32,22 +36,32 @@ Give two examples of jQuery getter methods and two examples of jQuery setter
 methods.
 
 <!-- your answer starts here -->
+Getters:
+document.body.style.backgroundColor
+$( 'p' ).html
 
+Setters:
+Document.body.style.backgroundColor = "red"
+$( 'li' ).html( 'Every list should have the same thing in it' )
```
Comment (benjimelito) [inline diagnostic.md:45]: Yes, we were looking for two examples of each though!

File: diagnostic.md [hunk 2]
```diff
@@ -65,5 +79,5 @@ body DOM node, then hides it.
 Use jQuery to register a click handler for clicks inside `div` items.
 
 <!-- your answer starts here -->
-
+$( '.div' ).click(function()
```
Comment (benjimelito) [inline diagnostic.md:82]: Make sure this syntax is complete

**Comment by benjimelito** [comment]
Nice work, 5/8

**Pull request closed without merging**